api/v1alpha1/virtualmachineclass_types.go

```diff
@@ -21,6 +21,31 @@ type DynamicDirectPathIODevice struct {
 	CustomLabel string `json:"customLabel,omitempty"`
 }
 
+// InstanceStorage encapsulates information about instance storage.
+type InstanceStorage struct {
+	// StorageClass describes the configuration of instance storage related attributes on the VirtualMachine instance.
```

Comment:
Could you expand the description to state what storageClass will be used if this field is omitted (I assume it's the per-namespace assigned one?)

Reply:
For example:

```
StorageClass refers to the name of a StorageClass resource used to
provide the storage for the configured, instance storage volumes.

The value of this field has no relationship to or bearing on the field
virtualMachine.spec.storageClass.

Please note the referred StorageClass must be available in the same
namespace as the VirtualMachineClass that uses it for configuring
instance storage.
```

Reply:
Yeah, I'm aware, but I also don't know how to best make the distinction since VM Classes *are* bound to specific namespaces. If that makes sense.